backend/protocol_rpc/validators_init.py

```
@@ -42,6 +87,27 @@ async def initialize_validators(
     if not isinstance(validators_data, list):
         raise ValueError("validators_json must contain a JSON array")
 
+    # Compare desired config against current DB state
+    try:
+        desired_hash = _desired_config_hash(validators_json)
+    except (TypeError, KeyError) as e:
```

Ruff (0.15.0)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)